src/Account/Service/UserService.spec.ts

```diff
@@ -0,0 +1,101 @@
+import UserService from './UserService';
+import { Test } from '@nestjs/testing';
+import { AppModule } from '../../AppModule';
+import BusinessError from '../../Common/BusinessError';
+import { ErrorCode } from '../../ErrorCode';
+import { HttpStatus, INestApplication } from '@nestjs/common';
+import { randomBytes } from 'crypto';
+import { RedisService } from 'nestjs-redis';
+import RedisKeys from '../../Common/RedisKeys';
+import * as moment from 'moment';
+import { Connection } from 'mongoose';
+import { getConnectionToken } from '@nestjs/mongoose';
+
+describe('UserService', () => {
+  let app: INestApplication;
+  let userService: UserService;
+  let redisService: RedisService;
+  let connection: Connection;
+  let createId;
+
+  const genMockUser = () => {
+    return {
+      name: 'testYser:' + randomBytes(5).toString('hex'),
+      dob: '12-12-2018',
+      description: randomBytes(10).toString('hex'),
+      address: randomBytes(10).toString('hex'),
+    };
+  };
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    app = moduleRef.createNestApplication();
+    userService = moduleRef.get<UserService>(UserService);
+    redisService = moduleRef.get<RedisService>(RedisService);
+    connection = moduleRef.get(getConnectionToken());
+  });
+
+  afterAll(async () => {
+    await redisService.getClient().flushdb();
+    await connection.dropDatabase();
+    await connection.close();
+    await app.close();
+  });
+
+  const testUser = genMockUser();
+
+  describe('UserService', () => {
+    it('should return id', async () => {
+      const result = await userService.createUser(testUser);
+
+      expect(result.id).not.toBe(null);
+      createId = result.id;
+    });
+
+    it('throw ResourceExist', async () => {
+      try {
+        await userService.createUser(testUser);
+      } catch (e) {
```

Comment:
the assertion fails if there is no catch at all.

Reply:
I also found this problem and it has been fixed
[Fix 21fe3db]https://github.com/Wiredcraft/test-backend/pull/62/commits/21fe3db8d849e3fdce6e410d59217f97310c2a7f